api-reference/beta/toc.yml

```
@@ -11166,6 +11166,63 @@ items:
             href: api/directory-deleteditems-restore.md
           - name: List deleted items owned by user
             href: api/directory-deleteditems-user-owned.md
+        - name: Delegated admin relationships
+          displayName: Partner Center, CSP, VAR
```

Additional suggestion to align this with the search terms @adtangir 
```suggestion
        - name: Granular delegated admin relationships
          displayName: Partner Center, GDAP, CSP, VAR
```